smart-contracts/test/Lock/partialWithdraw.js

```diff
@@ -0,0 +1,81 @@
+
+const Units = require('ethereumjs-units')
+
+const deployLocks = require('../helpers/deployLocks')
+const Unlock = artifacts.require('../Unlock.sol')
+let unlock, locks
+
+const Promise = require('bluebird')
+if (typeof web3.eth.getBlockPromise !== 'function') {
+  Promise.promisifyAll(web3.eth, { suffix: 'Promise' })
+}
+
+contract('Lock', (accounts) => {
+  before(() => {
+    return Unlock.deployed()
+      .then(_unlock => {
```

Comment:
no problem. I'll convert it to use async/await. What do you think about (in a separate PR) extracting the zos deployment process into a helper, and reusing in cases like this where we currently deploy Unlock without a proxy? It would help to keep our use of a proxy consistent in our tests.

Reply:
That is a good idea for another PR!